tests/functional/api/test_deploy_tokens.py

```diff
@@ -1,9 +1,13 @@
+from datetime import date
```

Comment:
nit: I'm personally not a fan of this type of import.

I prefer: `import datetime` and then just use `datetime.date.today()`